src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/Public.Data/BatchMappingUpdateViewModels.cs

```diff
@@ -21,29 +51,56 @@ public Validator()
     }
 }
 
-public record LocationMappingUpdateRequest
+public record BatchFilterOptionMappingUpdatesRequest
 {
-    public GeographicLevel? Level { get; init; }
+    public List<FilterOptionMappingUpdateRequest> Updates { get; init; } = [];
 
-    public string? SourceKey { get; init; } = null;
+    public class Validator : AbstractValidator<BatchFilterOptionMappingUpdatesRequest>
+    {
+        public Validator()
+        {
+            RuleForEach(request => request.Updates)
+                .SetValidator(new FilterOptionMappingUpdateRequest.Validator());
+        }
+    }
+}
+
+public record LocationMappingUpdateResponse : MappingUpdateResponse<LocationOptionMapping, MappableLocationOption>
+{
+    public GeographicLevel Level { get; init; }
+}
+
+public record FilterOptionMappingUpdateResponse : MappingUpdateResponse<FilterOptionMapping, MappableFilterOption>
+{
+    public string FilterKey { get; init; } = string.Empty;
+}
```

Comment:
I think these have come in a PR too early

Reply:
Yup it did somehow, whoops!  I've decided to close this PR and incorporate all changes into the PR for EES-5115 if that's OK.